Assets/PoisonMushroom.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PoisonMushroom : MonoBehaviour
```

Comment:
This should inherit from the EnvironmentHealth class so that the bullet and player health class can take care of the collisions